extensions/amp-mustache/0.1/test/test-amp-mustache.js

```diff
@@ -319,9 +318,12 @@ describe('amp-mustache template', () => {
       const nestedTemplateElement = outerResult.querySelector('template');
       const nestedTemplate = new AmpMustache(nestedTemplateElement);
       nestedTemplate.compileCallback();
+      console.error.returns(null);
```

Comment:
I think this line and the `expect` on line 325 should together be rewritten as a function in test/_init_tests.js that other tests can call. Basically, expose a function that temporarily stops `console.error` from throwing an exception.